2 - RTB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -665,6 +665,72 @@ Le decisioni vengono dunque così classificate:
   - implementative;
   - architetturali. \
 
+
+== Processo di Gestione dei Rischi
+_Conformant to outcomes to ISO/IEC/IEEE 12207:2017 clause 6.3.4_
+
+=== Scopo
+Lo scopo del processo di Gestione dei Rischi è identificare, analizzare, trattare e monitorare costantemente i rischi, così da poterli affrontare sistematicamente durante l'intero ciclo di vita del progetto.
+
+=== Risultati 
+Come risultato dell'implementazione del processo di Gestione dei Rischi:
+- vengono identificati i rischi;
+- i rischi vengono analizzati;
+- vengono identificate, priorizzate e selezionate opzioni di trattamento del rischio;
+- viene implementato un trattamento appropriato;
+- i rischi vengono valutati per verificare cambiamenti di stato e progressi nel trattamento.
+
+=== Attività e compiti
+Il progetto deve implementare le seguenti attività e compiti, in conformità con le norme identificate applicabili al processo di Gestione dei Rischi.
+
+==== Pianificare la gestione dei rischi  
+La strategia di gestione dei rischi per il progetto è basata su un approccio proattivo per identificare e mitigare i rischi in tutte le fasi del suo ciclo di vita. \
+I rischi devono venire individuati con tempestività, consentendo la pianificazione di misure preventive e correttive in tempo utile per limitare al massimo le conseguenze.\
+La classificazione dei rischi è basata sia sulla probabilità di occorrenza che sull'impatto che essi hanno sullo stato di avanzamento dei lavori e sul progetto stesso.
```

Comment:
come ti ho scritto dopo, aggiungerei qui il parametro delle conseguenze

Reply:
non sono d'accordo perchè le conseguenze in sè non sono un parametro, la loro gravità influenza il parametro "impatto".